ganga/GangaCore/Core/InternalServices/Coordinator.py

```diff
@@ -92,15 +92,9 @@ def _checker(): return True
 
 
 def disableMonitoringService():
-
-    # disable the mon loop
-    log.debug("Shutting down the main monitoring loop")
-    from GangaCore.Core.MonitoringComponent.Local_GangaMC_Service import _purge_actions_queue, stop_and_free_thread_pool
-    _purge_actions_queue()
-    stop_and_free_thread_pool()
     log.debug("Disabling the central Monitoring")
     from GangaCore.Core import monitoring_component
-    monitoring_component.disableMonitoring()
```

Comment:
Should this `disableMonitoring` method be removed?

Reply:
It is replaced by the `monitoring_component.disable()` method on the next line. Its new implementation is [here](https://github.com/ganga-devs/ganga/blob/6f23ac6c0cb9406d978d93a3993bfad5ebe331d1/ganga/GangaCore/Core/MonitoringComponent/MonitoringService.py#L94)